certbot/client.py

```diff
@@ -4,27 +4,24 @@
 import os
 import platform
 
-
+import OpenSSL
+import josepy as jose
+import zope.component
 from cryptography.hazmat.backends import default_backend
 # https://github.com/python/typeshed/blob/master/third_party/
 # 2/cryptography/hazmat/primitives/asymmetric/rsa.pyi
 from cryptography.hazmat.primitives.asymmetric.rsa import generate_private_key  # type: ignore
-import josepy as jose
-import OpenSSL
-import zope.component
 
 from acme import client as acme_client
 from acme import crypto_util as acme_crypto_util
 from acme import errors as acme_errors
 from acme import messages
-from acme.magic_typing import Optional  # pylint: disable=unused-import,no-name-in-module
+from acme.magic_typing import Optional  # type: ignore  # pylint: disable=unused-import,no-name-in-module
```

Comment:
I don't think this `type: ignore` is needed. I ran tests locally without it and they passed.

Reply:
I heavily relies on the amazing refactor capabilities of Pycharm here, to make wide refactoring, but sometimes, it makes some little weird glitches. Anyway, it is fixed.